docs/components/modeler/bpmn/bpmn.md

```diff
@@ -0,0 +1,46 @@
+---
+id: camunda-modeler-bpmn
+title: BPMN in Camunda Modeler
+---
+
+# Quickstart
+
+## Create new BPMN 2.0 Diagram
+
+![New BPMN Diagram](./img/quickstart-1.png)
+
+To start modeling, create a new BPMN 2.0 diagram for the Camunda Platform by selecting **Create diagram > Create new BPMN diagram (Camunda Platform)** in the top-level menu.
```

Comment:
This is not Cloud-first. Can you adjust the screenshot and the opening lines here? 

I would even advocate to remove the screenshot (or have it highlight Cloud) and modify the section to be tool agnostic.

```suggestion
![New BPMN Diagram](./img/quickstart-1.png)

To start modeling, create a new BPMN 2.0 diagram by selecting **Create diagram > Create new BPMN diagram** for your process engine in the top-level menu.
```

Reply:
Updated in latest commit, but note I am drafting the "Automating a process using BPMN" doc this week, which should replace much of this content.